CPT4/load_stage.sql

```diff
@@ -573,9 +358,25 @@ WHERE NOT EXISTS (
 			AND crs.relationship_id = 'Is a'
 			AND crs.vocabulary_id_1 = 'CPT4'
 			AND crs.vocabulary_id_2 = 'CPT4'
-		);
+		); -- 14292
+		
+--12. Add everything from the Manual tables
+DO $_$
+BEGIN
+     PERFORM VOCABULARY_PACK.ProcessManualConcepts();
+END $_$; -- OK
+
+DO $_$
+BEGIN
+     PERFORM VOCABULARY_PACK.ProcessManualSynonyms();
+END $_$;
+
+DO $_$
+BEGIN
+	PERFORM VOCABULARY_PACK.ProcessManualRelationships();
+END $_$;
 
---11. Extract all CPT4 codes inside the concept_name of other cpt codes.
+--13. Extract "hiden" CPT4 codes inside concept_names of another CPT4 codes.
```

Comment:
why do we run this if it gives 0?

Reply:
This point is from the old version of the load stage. I left it here just in case that this can happen again.